io.ossia.score.yml

```diff
@@ -0,0 +1,281 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+rename-desktop-file: ossia-score.desktop
+rename-icon: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Required for USB, Webcam devices support as well as raw GPIO I/O
+  - --device=all
+
+  # X11 is used by default
+  - --socket=x11
+
+  # Used by X11
+  - --share=ipc
+
+  # Network access
+  - --share=network
+
+  # PulseAudio is needed for ALSA device access
+  - --socket=pulseaudio
+
+  # Audio support through pipewire
+  - --filesystem=xdg-run/pipewire-0
+
+  # Ignore ALSA configuration
+  - --env=ALSA_CONFIG_PATH=
+
+  # Plugins
+  - --env=VST_PATH=/app/extensions/Plugins/vst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
```

Comment:
I'll reiterate what I said previously. This path for LV2 is incorrect because it doesn't include `~/.lv2` nor `/app/lib/lv2`. Unfortunately LV2 doesn't expand `~` and `$HOME` isn't expanded here.

Reply:
nevermind. just put it all in the wrapper script and remove it here.